mlflow/java/client/pom.xml

```diff
@@ -5,13 +5,13 @@
   <parent>
     <groupId>org.mlflow</groupId>
     <artifactId>mlflow-parent</artifactId>
-    <version>0.7.0</version>
+    <version>0.7.1</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
   <artifactId>mlflow-client</artifactId>
   <packaging>jar</packaging>
-  <version>0.7.0</version>
+  <version>0.7.1</version>
```

Comment:
Decided to not add the .dev suffix here just to be safe, though I can't find maven documentation that explicitly says  such suffixes are invalid (e.g. [this guide](https://maven.apache.org/guides/mini/guide-naming-conventions.html) recommends numeric version numbers but doesn't forbid non-numeric ones)